liquidctl/driver/smart_device.py

```diff
@@ -518,7 +518,10 @@ def parse_led_info(msg):
                     ret.append((f'LED {c + 1} accessory {a + 1}',
                                    Hue2Accessory(accessory_id), ''))
 
-        self._read_until({b'\x11\x01': parse_firm_info, b'\x21\x03': parse_led_info})
+        parsers = {b'\x11\x01': parse_firm_info}
+        if self._color_channels['sync'] > 0:
```

Comment:
The `sync` channel is kind of a hack.  So we probably shouldn't deal with it directly in too many places.

How about changing `__init__()` to only create the `sync` channel when it makes sense, and then elsewhere just doing `if self._color_channels` / `if not self._color_channels`?